app/views/relationships/create.js.erb

```diff
@@ -0,0 +1,2 @@
+$("#follow_form").html("<%= j render("users/unfollow") %>");
```

Comment:
1. Use single quote in js file 
2. Use double quote in rubyIn summary
3. Remove redundant (), in summary
```ruby
$('#follow_form').html('<%= j render "users/unfollow" %>');
```